app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
```

Comment:
It is good to have an infrastructure like

* controllers - When a request is made, controllers have the responsibility to handle them(In your case AnalyticsService)
* routes  - The endpoints (app.use(some endpoint))
* middlewares - this is a code executed before controllers. Example for this could be authorization middleware. Here you have `bodyparser` which is a typical middleware
* models - The db collections schema